mongodb.json

```diff
@@ -24,55 +25,55 @@
         "wheel": [
             {
                 "module": "certifi",
-                "input_file": "wheels/shared/certifi-2021.10.8-py2.py3-none-any.whl"
+                "input_file": "wheels/certifi-2021.10.8-py2.py3-none-any.whl"
             },
             {
                 "module": "chardet",
-                "input_file": "wheels/shared/chardet-3.0.4-py2.py3-none-any.whl"
+                "input_file": "wheels/chardet-3.0.4-py2.py3-none-any.whl"
             },
             {
                 "module": "idna",
-                "input_file": "wheels/shared/idna-2.10-py2.py3-none-any.whl"
+                "input_file": "wheels/idna-2.10-py2.py3-none-any.whl"
             },
             {
                 "module": "pymongo",
-                "input_file": "pymongo/pymongo-3.6.0-cp27-cp27mu-manylinux1_x86_64.whl"
+                "input_file": "wheels/pymongo-3.9.0-cp36-cp36m-manylinux1_x86_64.manylinux_2_5_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl"
```

Comment:
we can just delete the whole `pip3_dependencies` key since this is a py3 wheel file and won't work with py3 anyways. e.g.: https://github.com/splunk-soar-connectors/gsgmail/pull/10